ledfx/effects/math.py

```diff
@@ -24,6 +25,88 @@ def interpolate_pixels(pixels, new_length):
     return new_pixels
 
 
+# Copied from scipy (BSD 3-Clause "New" or "Revised" License) to avoid importing
```

Comment:
We need to include the actual license.txt here - https://github.com/scipy/scipy/blob/main/LICENSE.txt

And at the bottom of this function we can just add a generic `# End BSD-3 Licensed Code`